tests/stub/routing/test_routing_v4x4.py

```diff
@@ -846,17 +849,19 @@ def test_should_fail_when_writing_on_writer_that_returns_forbidden_on_read_only_
                 )
             failed = True
         session.close()
-        driver.close()
 
         self.assertNotIn(self._writeServer1.address,
                          driver.getRoutingTable(self.adb).writers)
+
```

Comment:
The blank line before `driver.close()` is there in some tests but not in others. I think I prefer having it there. You can also decide to not have it, but please be consistent.